src/DacpacTool/PackageBuilder.cs

```diff
@@ -39,14 +35,26 @@ public void AddExternalReference(string referenceFile, string externalParts)
             // Ensure that the model has been created
             EnsureModelCreated();
 
+            ValidateReference(referenceFile);
+
+            Console.WriteLine($"Adding reference to {referenceFile} with external parts {externalParts}");
+            Model.AddReference(referenceFile, externalParts);
+        }
+
+        private void ValidateReference(string referenceFile)
+        {
             // Make sure the file exists
             if (!File.Exists(referenceFile))
             {
                 throw new ArgumentException($"Unable to find reference file {referenceFile}", nameof(referenceFile));
             }
 
-            Console.WriteLine($"Adding reference to {referenceFile} with external parts {externalParts}");
-            Model.AddReference(referenceFile, externalParts);
+            // Make sure the file is a .dacpac file
+            var fileType = Path.GetExtension(referenceFile);
+            if (fileType != ".dacpac")
```

Comment:
Casing?

Reply:
Added a `ToLower()` to this, hopefully that's what you were getting at